Templates/Files/MS SQL/MS SQL database SELECT to CSV file/metadata.json

```diff
@@ -0,0 +1,12 @@
+{
+  "name": "MS SQL database SELECT to CSV file",
+  "version": "0.1.1",
+  "tags": [
+    "MS SQL",
```

Comment:
Remove "MS SQL", "SFTP", "JSON" and "CSV". Add tags "File Formats", "Protocols", "Databases".